// lib0/dist/buffer.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

require('./string-04e96f2d.cjs');
require('./environment-2789c99d.cjs');
require('./array-f5410bdd.cjs');
require('./math-08e068f9.cjs');
require('./encoding-62f41e96.cjs');
require('./decoding-9732ddf2.cjs');
var buffer = require('./buffer-cba90761.cjs');
require('./map-0dabcc55.cjs');
require('./conditions-f5c0c102.cjs');
require('./storage.cjs');
require('./function-6c17de82.cjs');
require('./object-491858d1.cjs');
require('./traits.cjs');
require('./set-a0a3ea69.cjs');
require('./number-466d8922.cjs');
require('./binary-ac8e39e2.cjs');
require('./error-8582d695.cjs');



exports.copyUint8Array = buffer.copyUint8Array;
exports.createUint8ArrayFromArrayBuffer = buffer.createUint8ArrayFromArrayBuffer;
exports.createUint8ArrayFromLen = buffer.createUint8ArrayFromLen;
exports.createUint8ArrayViewFromArrayBuffer = buffer.createUint8ArrayViewFromArrayBuffer;
exports.decodeAny = buffer.decodeAny;
exports.encodeAny = buffer.encodeAny;
exports.fromBase64 = buffer.fromBase64;
exports.fromBase64UrlEncoded = buffer.fromBase64UrlEncoded;
exports.fromHexString = buffer.fromHexString;
exports.shiftNBitsLeft = buffer.shiftNBitsLeft;
exports.toBase64 = buffer.toBase64;
exports.toBase64UrlEncoded = buffer.toBase64UrlEncoded;
exports.toHexString = buffer.toHexString;
